.github/workflows/ci.yaml

```
@@ -1,23 +1,32 @@
 name: Build/test
 on:
-  push:
-    branches:
-      - "master"
-  pull_request:
+    push:
+        branches:
+            - 'master'
+    pull_request:
 jobs:
-  test:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: true
-      matrix:
-        os: [macos-latest, ubuntu-latest]
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
-        with:
-          python-version: '3.11'
-      - uses: actions/setup-node@v3
-        with:
-          node-version: '18'
-      - run: npm install
-      - run: npm test
+    test:
+        runs-on: ${{ matrix.os }}
+        strategy:
+            fail-fast: true
+            matrix:
+                os: [macos-latest, ubuntu-latest]
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-python@v4
+              with:
+                  python-version: '3.11'
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: '18'
+            - run: npm install
+            - name: generate grammar
+              run: npm run build
+            - name: check for uncommited changes
+              run: git diff --exit-code
+            - run: npm test
```

Is it worth to add something like 
```suggestion
            - run: npm test
            - name: check test file formatting
              run: npx tree-sitter test -u && git diff --exit-code
```

---

Hmm, shouldn't it have failed here :thinking: ?

https://github.com/ngalaiko/tree-sitter-go-template/actions/runs/12795223343/job/35672111612?pr=28

---

I think once you rebase that should fail

---

I think once you rebase that should fail. Tested on master and it failed for me.

---

Nope, I guess `test -u` will only reformat the files if at least one test changes.

https://github.com/ngalaiko/tree-sitter-go-template/actions/runs/12835719613/job/35795764049

---

Yeah, I must have changed something when testing. This behaviour is a bit weird. I will look into it upstream.